### DISCUSSION TITLE
How to be prepared for Upcoming Tea Testnet on 21 Feb 2024

### DISCUSSION THREAD

**Discussion by rav2525** [teaxyz/white-paper#138] [General]
For Non-developers:
https://tea.xyz/blog/10-ways-for-non-devs-to-participate-in-itn?utm_source=twitter&utm_medium=organic

For Developers:
https://tea.xyz/blog/tea-protocol-incentivized-testnet

**Comment by hamaadh** [comment]
Tahnks for sharing this information from the right sources 

Discord: hamadah6595

**Reply by rav2525** [reply]
You can connect your git profile in Discord settings, may not need to add your discord handle everywhere.

**Reply by hamaadh** [reply]
> You can connect your git profile in Discord settings, may not need to add your discord handle everywhere.

Already connected, thanks

**Reply by rav2525** [reply]
cool

**Reply by faudaz** [reply]
nice

**Comment by mrn1ck13** [comment]
Yes I also just connected my Github Account through the Discord app. Just have to click on the scroll wheel on the bottom right of the discord app and click on the GitHub app and connect through there.

**Comment by gentrazor** [comment]
Insightful

**Comment by rav2525** [comment]
5 days to go!

**Comment by hanriiver** [comment]
good

**Comment by rav2525** [comment]
1 day to go for testnet

**Reply by etherbiln** [reply]
Just 2h to go for testnet!
